cmd/der2ascii/writer_test.go

```diff
@@ -131,12 +131,24 @@ var derToASCIITests = []convertFuncTest{
 	// BIT STRINGs are encoded normally if the contents are not an element.
```

Comment:
Probably worth revising these comments because "normally" could mean one of two things now. (Though see comment about elements vs. short bit string ordering. We should resolve that first.)

Reply:
s/normally/bit string literals, does that catch what you meant?